twitch-chat/index.js

```diff
@@ -88,6 +88,18 @@ function update() {
 
 }
 
+app.use(express.json());
+
+app.use('*', function(req, res, next) {
```

Comment:
What's all this about? What feature needs this?

Reply:
To add the headers to allow outside sources in themes. Because content security policy is a b